### DISCUSSION TITLE
用 docker 部署时，初始账号密码是什么？

### DISCUSSION THREAD

**Discussion by ZhangHouhou** [usememos/memos#89] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/39576967/175768709-db74c584-5466-4ceb-80f2-f48d2a790f6b.png)

**Comment by boojack** [comment, marked as answer]
可以参考下这个：https://github.com/usememos/memos/issues/87
* 如果 mode 是 dev，则会自动填充账号密码；
* 如果 mode 是 prod，则初始化数据库后，第一次进入需要输入管理员的账号密码；

**Reply by lxxself** [reply]
参看说prod第一次输入的账号密码就是登录用，但我部署完直接打开：ip:5230 就是跳到登录页，一直输入需要设置的用户名密码也不对，你说的管理员的账号密码是指什么？麻烦明示指教下，谢谢

**Reply by boojack** [reply]
感谢反馈，复现了，这几天会尝试修复

**Reply by boojack** [reply]
Fixed https://github.com/usememos/memos/issues/90#issuecomment-1167226300

**Reply by beyondtop** [reply]
我发现有个情况，就是第一次设置好管理员后，后面再登陆，输入用户名和密码，如果习惯性的的按回车，此时是无反应的，再用鼠标点击提交，必定提示401，无法正常登陆。1.3和dev好像都有这个问题。

**Comment by leoneoo** [comment]
群辉使用docker配置 prod版本，第一次登录，能否使用自己邮箱？貌似只给了demo@usememos.com 账户。虽然能登录，但是总感觉有点奇怪。能否删除这个demo账户呢？

**Reply by boojack** [reply]
感觉是启动命令的问题，可以先确定是不是 prod，在 prod 模式下是不会出现 demo 用户的。

**Reply by Rookiewan** [reply]
@leoneoo 环境变量问题，群晖如果用自带 docker 管理面板，可以在创建容器时，切换到命令，添加 --mode prod --port 5230
